app/src/androidTest/java/au/com/shiftyjelly/pocketcasts/deeplink/DeepLinkFactoryTest.kt

```diff
@@ -245,14 +245,25 @@ class DeepLinkFactoryTest {
     }
 
     @Test
-    fun pocketCastsWebsiteDeepLink() {
+    fun pocketCastsWebsiteGetDeepLink() {
         val intent = Intent()
             .setAction(ACTION_VIEW)
-            .setData(Uri.parse("https://pocketcasts.com"))
+            .setData(Uri.parse("https://pocketcasts.com/get"))
 
         val deepLink = factory.create(intent)
 
-        assertEquals(PocketCastsWebsiteDeepLink, deepLink)
+        assertEquals(PocketCastsWebsiteGetDeepLink, deepLink)
+    }
+
+    @Test
+    fun pocketCastsWebsiteGetSlashDeepLink() {
```

Comment:
Nice! Could you also add a test for a link with query parameters? Context: https://github.com/Automattic/pocket-casts-android/pull/2559

Reply:
And maybe another one for a referral link without the code?